runtime/queries/tsq/injections.scm

```diff
@@ -1,8 +1,10 @@
 ((comment) @injection.content
- (#set! injection.language "comment"))
+  (#set! injection.language "comment"))
 
 ((predicate
-   (predicate_name) @_predicate
-   (string) @injection.content)
- (#eq? @_predicate "#match?")
- (#set! injection.language "regex"))
+  name: (identifier) @_name
+  parameters:
+    (parameters
+      (string) @injection.content))
+  (#any-of? @_name "match" "not-match" "vim-match" "not-vim-match")
```

Comment:
```suggestion
  (#any-of? @_name "match" "not-match")
```

the vim ones are specific to neovim, I believe they use a different regex flavor

Reply:
Yes, but we can still highlight these function names the same.